GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Utils/Server.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace Utils
```

Comment:
Either remove the namespace, or apply namespace to all classes.

Reply:
Yep, going to extract the functions from gameDataController and put it in NetworkRequestController.